src/searchForm/overlay/index.tsx

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components'
+
+import { componentSizes } from '_utils/branding'
+
+import Overlay, {
+  enterTransitionDuration,
+  exitTransitionDuration,
+  reducedMotionTransitionDuration,
+  TRANSITION_CLASS_NAME,
+} from './Overlay'
+
+const transitionTimingFunction = 'cubic-bezier(0.2, 1, 0.2, 1)'
+
+const StyledOverlay = styled(Overlay)`
+  & .${TRANSITION_CLASS_NAME}-enter {
+    opacity: 0;
+    width: 0;
+  }
+
+  & .${TRANSITION_CLASS_NAME}-enter-active {
+    opacity: 1;
+    width: ${componentSizes.searchOverlayWidth};
+    transition: width ${enterTransitionDuration}ms ${transitionTimingFunction},
```

Comment:
does styled-component produce vendor specific transition rules ?

Reply:
I don't think we need any
https://caniuse.com/#feat=css-transitions

Reply:
I checked it's oupouting the code with prefixes.